src/OrleansCodeGenerator/Boostrap.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace OrleansCodeGenerator
+{
+    using Orleans.CodeGeneration;
+
+    public class Boostrap : IRuntimeCodeGenerator
```

Comment:
As I understand, the purpose of Bootstrap class is to have a non-static class that can be dynamically instantiated on the client side via Reflection, correct? A more explicit name (`RuntimeCodeGenerator `? `RuntimeCodeGeneratorWrapper`?) and a comment would help explain that, especially because the class isn't referenced anywhere.

The alternative is to use `GrainClient.InvokeStaticMethodThroughReflection` instead but I like your approach better.